config_defaults.yml

```
@@ -640,3 +640,5 @@ spec/spec_helper.rb:
   unmanaged: true
 .github/workflows/auto_release.yml:
   unmanaged: true
+.github/workflows/spec.yml:
```

shouldn't this be `unmanaged:true`, too?
```suggestion
.github/workflows/spec.yml:
  unmanaged: true
```